psydac/linalg/tests/test_linalg.py

```diff
@@ -0,0 +1,959 @@
+import pytest
+import numpy as np
+
+from psydac.linalg.block import BlockLinearOperator, BlockVector, BlockVectorSpace
+from psydac.linalg.basic import ZeroOperator, IdentityOperator, ComposedLinearOperator, SumLinearOperator, PowerLinearOperator, InverseLinearOperator, ScaledLinearOperator
+from psydac.linalg.stencil import StencilVectorSpace, StencilVector, StencilMatrix
+from psydac.linalg.solvers import BiConjugateGradient, ConjugateGradient, PConjugateGradient, inverse
+from psydac.ddm.cart         import DomainDecomposition, CartDecomposition
+#===============================================================================
+
+# 11.12.22: Check for redundancy(?) in pytests 1,2 and 4, document 1,2
```

Comment:
Have you checked?

Reply:
No, good catch. Did so now, will be changed with the next commit.